src/Windows/Avalonia.Win32/WintabImpl/WintabInfo.cs

```diff
@@ -0,0 +1,505 @@
+            }
+
+            CMemUtils.FreeUnmanagedBuf(buf);
+
+            return axis;
+        }
+
+        /// <summary>
+        /// Return the number of tablets that have at some time been attached.
+        /// A record of these devices is in the tablet settings.  Since there
+        /// is no direct query for this value, we have to enumerate all of
+        /// the tablet settings.
+        /// </summary>
+        /// <returns>tablet count</returns>
+        public static UInt32 GetNumberOfConfiguredDevices()
+        {
+            UInt32 numConfiguredTablets = 0;
+            try
+            {
+                WintabLogContext ctx = new WintabLogContext();
+                IntPtr buf = CMemUtils.AllocUnmanagedBuf(ctx);
+
+                for (Int32 idx = 0; idx < MAX_NUM_ATTACHED_TABLETS; idx++)
+                {
+                    int size = (int)CWintabFuncs.WTInfoA(
+                        (UInt32)(EWTICategoryIndex.WTI_DDCTXS + idx), 0, buf);
+                    if (size == 0)
+                    {
+                        break;
+                    }
+                    else
+                    {
+                        numConfiguredTablets++;
+                    }
+                }
+
+                CMemUtils.FreeUnmanagedBuf(buf);
+            }
+            catch (Exception ex)
+            {
+                MessageBox.Show("FAILED GetNumberOfConfiguredDevices: " + ex.ToString());
+            }
+
+            return numConfiguredTablets;
+        }
+
+        /// <summary>
+        /// Returns a list of indecies of previous or currently attached devices.
+        /// It is up to the caller to use the list to determine which devices are
+        /// actually physically device by responding to data events for those devices.
+        /// Devices that are not physically attached will, of course, never send
+        /// a data event.
+        /// </summary>
+        /// <returns></returns>
+        public static List<Byte> GetFoundDevicesIndexList()
+        {
+            List<Byte> list = new List<Byte>();
+
+            try
+            {
+                WintabLogContext ctx = new WintabLogContext();
+                IntPtr buf = CMemUtils.AllocUnmanagedBuf(ctx);
+
+                for (Int32 idx = 0; idx < MAX_NUM_ATTACHED_TABLETS; idx++)
+                {
+                    int size = (int)CWintabFuncs.WTInfoA(
+                        (UInt32)(EWTICategoryIndex.WTI_DDCTXS + idx), 0, buf);
+                    if (size == 0)
+                    {
+                        break;
+                    }
+                    else
+                    {
+                        list.Add((Byte)idx);
+                    }
+                }
+
+                CMemUtils.FreeUnmanagedBuf(buf);
+            }
+            catch (Exception ex)
+            {
+                MessageBox.Show("FAILED GetNumberOfConfiguredDevices: " + ex.ToString());
+            }
+
+            return list;
+        }
+    }*/
```

Comment:
I left this from original Wacom sample implementation, in case it is needed later. I can remove it if needed